Funcky.Test/Sequence/ConcatTest.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Immutable;
+using System.Linq;
+using FsCheck;
+using FsCheck.Xunit;
+using Xunit;
+
+namespace Funcky.Test
+{
+    public sealed class ConcatTest
+    {
+        [Fact]
+        public void ConcatenatedSequenceIsEmptyWhenNoSourcesAreProvided()
+        {
+            Assert.Empty(Sequence.Concat(Enumerable.Empty<object>()));
```

Comment:
Isn't that an empty source instead of no source? Has Concat an overload which is empty? Would that be useful?

Reply:
Indeed params also allows to call the function without arguments.

Reply:
6a43bbf31e30a9edda241eef5f9491225b1cd877